lib/stages/hexDump.cpp

```diff
@@ -55,6 +61,11 @@ void hexDump::main_thread() {
         // Prints the hex data to screen
         hex_dump(16, (void*)&frame[_offset], _len);
 
+        // Add sample to trackers
+        static int cnt = 0;
```

Comment:
I would avoid using a static here.  With more than one stage, this will have possibly unexpected results.   If you just move this declaration outside the `while`, then it will achieve the same effect.